static/app/components/feedback/feedbackItem/feedbackItem.tsx

```diff
@@ -19,48 +24,37 @@ import TextCopyInput from 'sentry/components/textCopyInput';
 import {IconJson, IconLink} from 'sentry/icons';
 import {t} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
-import {Event, GroupStatus} from 'sentry/types';
+import type {Event} from 'sentry/types';
+import {GroupStatus} from 'sentry/types';
 import type {FeedbackIssue} from 'sentry/utils/feedback/types';
-import useApi from 'sentry/utils/useApi';
 import useOrganization from 'sentry/utils/useOrganization';
 
 interface Props {
   eventData: Event | undefined;
   feedbackItem: FeedbackIssue;
-  refetchIssue: () => void;
   tags: Record<string, string>;
 }
 
-export default function FeedbackItem({
-  feedbackItem,
-  eventData,
-  refetchIssue,
-  tags,
-}: Props) {
+export default function FeedbackItem({feedbackItem, eventData, tags}: Props) {
   const organization = useOrganization();
   const hasReplayId = useFeedbackHasReplayId({feedbackId: feedbackItem.id});
   const {markAsRead, resolve} = useMutateFeedback({
-    feedbackId: feedbackItem.id,
+    feedbackIds: [feedbackItem.id],
     organization,
-    refetchIssue,
   });
-  const api = useApi();
-
-  const markReadUrl = `/organizations/${organization.slug}/issues/${feedbackItem.id}/`;
-
-  useEffect(() => {
-    (async () => {
-      await api.requestPromise(markReadUrl, {
-        method: 'PUT',
-        data: {hasSeen: true},
-      });
-      refetchIssue();
-    })();
-  }, []); // eslint-disable-line
```

Comment:
moved this into `useFetchFeedbackData()`. Also add a comment because it's explicitly after the initial fetch.